libs/components/modals/src/lib/modules/modal/modal.component.scss

```diff
@@ -175,6 +175,13 @@
 
   .sky-modal-content {
     padding: 0;
+
+    ::ng-deep .sky-tabset {
```

Comment:
Would it be possible to put this override in the tabset's SCSS file using `:host-context(.sky-modal)`? My reasoning: we should isolate component-specific overrides in that component's source directory so that they're easy to find later on.